demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -68,7 +68,8 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
     }
     
     private func initSDK(result: @escaping FlutterResult) {
-        kms = LocalkmsNewKMS(nil, nil)
+        let memKMSStore = LocalkmsNewMemKMSStore()
+        kms = LocalkmsNewKMS(memKMSStore, nil)
```

Comment:
I will be changing to pass ios userdefault store right ??

Reply:
Yes :) (your implementation of that interface)